src/spyglass/sharing/sharing_kachery.py

```diff
@@ -215,3 +215,38 @@ def download_file(analysis_file_name: str) -> bool:
             raise Exception(f"{analysis_file_name} cannot be downloaded")
 
         return True
+
+
+def share_data_to_kachery(
+    restriction={},
+    table_list=[],
+    zone_name="franklab.default",
+):
+    """Share data to kachery
+
+    Parameters
+    ----------
+    restriction : dict, optional
+        restriction to select what data should be shared from table, by default {}
+    table_list : list, optional
+        List of tables to share data from, by default []
+    zone_name : str, optional
+        What kachery zone to share the data to, by default "franklab.collaborators"
+
+    Raises
+    ------
+    ValueError
+        Does not allow sharing of all data in table
+    """
+    kachery_selection_key = {"kachery_zone_name": zone_name}
+    if not restriction:
+        raise ValueError("Must provide a restriction to the table")
+    for table in table_list:
+        analysis_file_list = (table & restriction).fetch("analysis_file_name")
+        for file in analysis_file_list:  # Add all analysis to shared list
+            kachery_selection_key["analysis_file_name"] = file
+            AnalysisNwbfileKacherySelection.insert1(
+                kachery_selection_key, skip_duplicates=True
+            )
+    AnalysisNwbfileKachery.populate()
+    return
```

Comment:
Return is unnecessary here. If there's no return, the function automatically returns None.